src/python/grpcio/grpc/_server.py

```diff
@@ -591,7 +591,13 @@ def _handle_call(rpc_event, generic_handlers, thread_pool,
     if not rpc_event.success:
         return None, None
     if rpc_event.request_call_details.method is not None:
-        method_handler = _find_method_handler(rpc_event, generic_handlers)
+        try:
+            method_handler = _find_method_handler(rpc_event, generic_handlers)
+        except Exception as e:  # pylint: disable=broad-except
```

Comment:
Name the Exception `exception`.

Reply:
Done (for the entire file, hence a separate commit)